routes/admin.php

```diff
@@ -0,0 +1,7 @@
+<?php
+use App\Http\Controllers\Admin\UserController;
+use App\Http\Controllers\Admin\PostController;
+use Illuminate\Support\Facades\Route;
+
+Route::resource('users', UserController::class);
+Route::get('user/{user}/posts', [PostController::class, "index"])->name("users.posts.index");
```

Comment:
Tham khảo cái này làm nested resouce cho đúng
https://laraveldaily.com/nested-resource-controllers-and-routes-laravel-crud-example/